posthog/async_migrations/migrations/0002_events_sample_by.py

```diff
@@ -120,12 +131,7 @@ def operations(self):
                     ON CLUSTER {CLICKHOUSE_CLUSTER}
                 """,
             ),
-            AsyncMigrationOperation.simple_op(
-                database=AnalyticsDBMS.CLICKHOUSE,
-                sql=f"OPTIMIZE TABLE {EVENTS_TABLE_NAME} FINAL",
-                rollback="",
-                resumable=True,
-            ),
+            AsyncMigrationOperation.simple_op(database=AnalyticsDBMS.CLICKHOUSE, fn=optimize_table_fn, rollback=""),
```

Comment:
This isn't `simple_op` anymore & add something as `debug_context` too, I just landed that PR.
```suggestion
            AsyncMigrationOperation(fn=optimize_table_fn),
```